apps/api-extractor/src/analyzer/Span.ts

```diff
@@ -404,39 +431,69 @@ export class Span {
   }
 
   /**
-   * Recursive implementation of `getModifiedText()` and `writeModifiedText()`.
+   * Returns a diagnostic dump of the tree, showing the SpanModification settings for each nodde.
    */
-  private _writeModifiedText(options: IWriteModifiedTextOptions): void {
-    if (this.modification.indentDocComment) {
-      if (options.indentDocCommentState !== IndentDocCommentState.Inactive) {
-        throw new InternalError('indentDocComment cannot be nested');
+  public getModifiedDump(indent: string = ''): string {
+    let result: string = indent + ts.SyntaxKind[this.node.kind] + ': ';
+
+    if (this.prefix) {
+      result += ' pre=[' + this._getTrimmed(this.modification.prefix) + ']';
+    }
+    if (this.suffix) {
+      result += ' suf=[' + this._getTrimmed(this.modification.suffix) + ']';
+    }
+    if (this.separator) {
+      result += ' sep=[' + this._getTrimmed(this.separator) + ']';
+    }
+    if (this.modification.indentDocComment !== IndentDocCommentScope.None) {
+      result += ' indentDocComment=' + IndentDocCommentScope[this.modification.indentDocComment];
```

Comment:
Just clarifying this doesn't need to be wrapped in `[...]` or `"..."`

Reply:
` IndentDocCommentScope[]` will return a text string like `indentDocComment=PrefixOnly` ✔